tensorflow_recommenders_addons/dynamic_embedding/__init__.py

```diff
@@ -37,13 +38,16 @@
     'enable_inference_mode',
     'enable_train_mode',
     'get_model_mode',
+    'trainable_wrapper_filter',
     'keras',
     'math',
+    'data_flow',
     'shadow_ops',
 ]
 
 from tensorflow_recommenders_addons.dynamic_embedding.python import keras
 from tensorflow_recommenders_addons.dynamic_embedding.python.ops import math_ops as math
+from tensorflow_recommenders_addons.dynamic_embedding.python.ops import data_flow_ops as data_flow
```

Comment:
Better to keep the `data_flow_ops` naming. This is consistent with the original naming of tf

Reply:
But "math_ops"……I thought removing "_ops" from the name would be more consistent with the pre-TFRA code.